prov/sockets/src/sock_ep_msg.c

```diff
@@ -1236,3 +1256,158 @@ int sock_msg_passive_ep(struct fid_fabric *fabric, struct fi_info *info,
 	return ret;
 }
 
+static void sock_ep_cm_check_msg_list(struct sock_ep_cm_head *cm_head)
```

Comment:
Please rename call to indicate what is happening.  I.e. we're processing a close_list

Reply:
Done